integration/spark-common-test/src/test/scala/org/apache/carbondata/datamap/lucene/LuceneFineGrainDataMapSuite.scala

```diff
@@ -733,6 +732,26 @@ class LuceneFineGrainDataMapSuite extends QueryTest with BeforeAndAfterAll {
     sql("DROP TABLE table1")
   }
 
+  test("test lucene fine grain empty string") {
+    sql("DROP TABLE IF EXISTS datamap_test_01")
+    CarbonProperties.getInstance().addProperty(CarbonCommonConstants.CARBON_LUCENE_EMPTY_STRING_ENABLE,"null")
+    sql(
+      "create table datamap_test_01(id int,fn string,ln string)stored by 'carbondata' " +
```

Comment:
use `stored as carbondata` and move to next line. please make it nice formated and captital